calculator.js

```diff
@@ -64,7 +64,7 @@
             if (isNaN(history[history.length - 1])) return;
             if (operation === "" || oldInput === "") return;
             if (operation === "+") {
-                result = +oldInput + +newInput;
+                result = +oldInput + (+newInput);

```

Comment:
Put () around +newInput because JSHint recommends we do this as it is confusing to have 2 + signs next to each other